src/Microsoft.DotNet.Cli.Utils/CommandResolution/PublishedPathCommandResolver.cs

```diff
@@ -0,0 +1,72 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.IO;
+
+namespace Microsoft.DotNet.Cli.Utils
+{
+    public class PublishedPathCommandResolver : ICommandResolver
```

Comment:
This doesn't actually have to be a "published" app right?  This would work against just a "built" app.

I'm wondering if "Published" is a confusing word here.  What exactly is the intention of the class?